### ISSUE TITLE
FR - Plugin manager

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3224]
Please add filter by compability 

By the way
sorting by compatibility doesn't work - it displays nothing
filtering doesn't work

**Comment by SimSync** [comment]
Yes, unfortunatly, the compatibility is not saved to the plugins table. It is read directly from the plugin.xml.
That means, filtering and sorting can't work.
@CaMer0n I think it would be a good idea to place also the compatibility information into the plugin table. Filtering and sorting for this information is something i stumbled upon already by myself.
What do you think?

**Comment by Jimmi08** [comment]
Just note: 
I am not sure now if compability is field on addons table on e107.org,  but there should be 2 records for one plugin - version compatible with v1 and version compatible with v2.  People should be still able download latest old version.
